commands/root.go

```diff
@@ -87,8 +88,10 @@ func addCommands(cmd *cobra.Command, dockerCli command.Cli) {
 		imagetoolscmd.RootCmd(dockerCli, imagetoolscmd.RootOptions{Builder: &opts.builder}),
 	)
 	if isExperimental() {
+		cmd.AddCommand(debugcmd.RootCmd(dockerCli, func(dops debugcmd.DebugOptions) *cobra.Command {
+			return buildCmd(dockerCli, opts, dops)
```

Comment:
We are just debugging the `build` cmd so sounds good to have `buildx debug` defaulting to build atm but could be any root cmd in the future such as `bake` with `buildx debug bake` so we might consider passing `cmd` instead and have a "debug" interface for each command supporting debug. WDYT?

Reply:
Thanks for the suggestion! It sounds good to me. I've fixed the patch to pass `*cobra.Command` with the debugger interface to `debugcmd.RootCmd`. PTAL :pray: